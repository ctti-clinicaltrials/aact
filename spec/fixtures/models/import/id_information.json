{
  "protocolSection": {
    "identificationModule": {
      "nctId": "NCT001",
      "nctIdAliases": [
        "NCT002"
      ],
      "orgStudyIdInfo": {
        "id": "NRG-GY032"
      },
      "secondaryIdInfos": [
        {
          "id": "NCI-2023-09355",
          "type": "REGISTRY",
          "domain": "CTRP"
        },
        {
          "id": "U10CA180868",
          "type": "NIH",
          "link": "https://reporter.nih.gov/quickSearch/U10CA180868"
        }
      ],
      "organization": {},
      "briefTitle": "Tailoring Therapy in ...",
      "officialTitle": "A Phase II Study of Tailored Adjuvant Therapy..."
    },
    "statusModule": {}
  }
}