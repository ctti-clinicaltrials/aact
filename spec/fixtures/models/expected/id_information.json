{
  "IdInformation": [
    {
      "nct_id": "NCT001",
      "id_source": "nct_alias",
      "id_value": "NCT002",
      "id_type": null,
      "id_type_description": null,
      "id_link": null
    },
    {
      "nct_id": "NCT001",
      "id_source": "org_study_id",
      "id_value": "NRG-GY032",
      "id_type": null,
      "id_type_description": null,
      "id_link": null
    },
    {
      "nct_id": "NCT001",
      "id_source": "secondary_id",
      "id_value": "NCI-2023-09355",
      "id_type": "REGISTRY",
      "id_type_description": "CTRP",
      "id_link": null
    },
    {
      "nct_id": "NCT001",
      "id_source": "secondary_id",
      "id_value": "U10CA180868",
      "id_type": "NIH",
      "id_type_description": null,
      "id_link": "https://reporter.nih.gov/quickSearch/U10CA180868"
    }
  ]
}